File: Data Structures & Algorithms/is-palindrome/submission-0.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.split(' ')
        s = ''.join(s)
        s = s.lower()
        s = list(s)
        i = 0
        while i < len(s):
            if not s[i].isalnum():
                s.remove(s[i])
            else:
                i = i+1
        s = ''.join(s)
        i = 0
        j = len(s)-1
        while i <j  :
            if s[i] != s[j]:
                return False
            else:
                i = i+1
                j = j-1
        return True

            
   
        

File: Data Structures & Algorithms/is-anagram/submission-0.js
class Solution {
    /**
     * @param {string} s
     * @param {string} t
     * @return {boolean}
     */
    isAnagram(s, t) {
        let inputOne = s.split('').sort().join('')
        let inputTwo = t.split('').sort().join('')
        if(s.length === t.length && inputOne===inputTwo){
            return true
        }else{
            return false
        }
    }
}
